### PR TITLE
Automate docs sync for KKP components page

### PR DESCRIPTION
**Pull request by embik**
**What this PR does / why we need it**:
The components page in our docs is outdated quite a bit. This adds the page to be synced from our docs.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind documentation

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: hack/ci/update-docs.sh [hunk 1]
```diff
@@ -61,6 +61,26 @@ make runbook
...
++++
+title = "KKP Components"
+date = 2021-04-13T20:07:15+02:00
+weight = 2
+
++++
+
+## Kubermatic Kubernetes Platform Components
+
+The following list is only eligible for the version that is currently available. Kubermatic has a strong emphasis on the security and reliability of the provided software and releases updates regularly that also update components.
```
Comment (xrstf) [inline hack/ci/update-docs.sh]: maybe "applicable" is better than "eligible". Eligible sounds like "available to be elected in a vote".

File: hack/ci/update-docs.sh [hunk 2]
```diff
@@ -61,6 +61,26 @@ make runbook
...
+
+| KKP Components                | Version                      |
+| ----------------------------- | ---------------------------- |
+EOT
+
+for filepath in $(find charts -name Chart.yaml | sort); do echo "| $(echo ${filepath} | sed -e 's/^\.\.\/charts\///g' -e 's/\/Chart\.yaml$//g') | $(yq '.appVersion' ${filepath} | sed -e 's/^v//g') |" >> ${components_file}; done
```
Comment (xrstf) [inline hack/ci/update-docs.sh]: I think your cat sneezed on your keyboard.

Comment (embik) [reply hack/ci/update-docs.sh]: I'm open to improvement suggestions 🤷🏻

**Comment by ahmedwaleedmalik** [comment]
/hold
Temporarily putting this on hold to make way for https://github.com/kubermatic/kubermatic/pull/11665

**Comment by ahmedwaleedmalik** [comment]
/unhold

👷‍♂️

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**